R/axis.R

```diff
@@ -14,12 +14,11 @@
 #' @rdname axis
 #' @export
 #' @examples
-#' 
 #' gs <- gsplot() %>%
 #'    points(x=1:5, y=1:5, legend.name="Stuff") %>%
 #'    lines(2:6, y=2:6, ylim=c(0,10)) %>%
 #'    axis(side=c(3,4),labels=FALSE) %>%
-#'    legend(location="topright")
+#'    legend("topright")
```

Comment:
This example somehow got screwed up in merge conflict resolution.  I thought I had everything from you, but not so.